scripts/pyFAI-drawmask

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#    Project: Azimuthal integration
+#             https://github.com/kif/pyFAI
+#
+#    Copyright (C) European Synchrotron Radiation Facility, Grenoble, France
+#
+#    Authors: Jérôme Kieffer (Jerome.Kieffer@ESRF.eu)
+#             V. Aramdo Solé <sole@esrf.fr>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+pyFAI-drawmask
+
+Use silx or PyMca module to define a mask
+"""
+
+__authors__ = ["Jerome Kieffer", "Valentin Valls"]
+__contact__ = "Jerome.Kieffer@ESRF.eu"
+__license__ = "GPLv3+"
```

Comment:
One comment: when a file has been modified only by the silx-team (full ESRF copyright) it would be great to switch it to MIT license and declare the file in copyright accordingly.